hooks/post/10-post_install.rb

```diff
@@ -35,9 +35,9 @@ def new_install?
 
     if Kafo::Helpers.module_enabled?(@kafo, 'katello')
       say <<MSG
-  * To install additional capsule on separate machine continue by running:
+  * To install an additional capsule on a separate machine run the following command:
 
-      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "<%= color('~/$CAPSULE-certs.tar', :info) %>"
+      capsule-certs-generate --capsule-fqdn "<%= color('$CAPSULE', :info) %>" --certs-tar "/root/certs.tar.gz"
```

Comment:
I don't think this should be changed, except maybe giving a path. Running the command you're giving now, they'll overwrite any other certs tarballs they've created.  It should still have $CAPSULE in the name.